NOTICE.txt

```diff
@@ -78,105 +80,82 @@ THE SOFTWARE.
 
 ## electron-context-menu
 
-This product contains a modified portion of 'electron-context-menu', which provides a context menu for Electron apps, by Sindre Sorhus.
+This product contains 'electron-context-menu' by Sindre Sorhus.
+
+Context menu for your Electron app
 
 * HOMEPAGE:
-  * https://github.com/sindresorhus/electron-context-menu
+  * https://github.com/sindresorhus/electron-context-menu#readme
```

Comment:
Curious why `#readme` is attached as part of the URL (similarly in a few other places)